apps/init.sh

```diff
@@ -4,24 +4,30 @@ set -e
 
 kubectl create namespace horizontal-scaling
 kubectl create namespace vertical-scaling
-
+kubectl create ns otel-demo
+kubectl label namespace  default oneagent=false
+kubectl label namespace otel-demo  oneagent=false
 ######################
 ### metrics-server ###
 ######################
 
 # https://artifacthub.io/packages/helm/metrics-server/metrics-server
-helm repo add metrics-server https://kubernetes-sigs.github.io/metrics-server/
-helm install metrics-server metrics-server/metrics-server \
-  --version 3.12.1 \
-  --namespace horizontal-scaling \
-  --values apps/metrics-server.values.yaml \
-  --atomic --wait
-
+#helm repo add metrics-server https://kubernetes-sigs.github.io/metrics-server/
+#helm install metrics-server metrics-server/metrics-server \
+ # --version 3.12.1 \
+  #--namespace horizontal-scaling \
+  ##--values apps/metrics-server.values.yaml \
+  #--atomic --wait
+kubectl apply -f https://github.com/kubernetes-sigs/metrics-server/releases/latest/download/components.yaml
```

Comment:
Let's user specific versions instead of `latest`. Like that we can make sure that the demo is always running. I still need to configure Renovate to keep them up to date though.

And the apply command is missing the namespace